sql/src/main/java/io/crate/planner/ResultDescription.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.planner;
+
+import java.util.Collection;
+
+/**
+ * Describes where a result resides and in which form
+ */
+public interface ResultDescription {
+
+    /**
+     * @return the nodeIds of the nodes that this result is on.
+     * If it's empty it is on the handler node.
+     */
+    Collection<String> executionNodes();
```

Comment:
Currently some phases implement both ResultDescription and ExecutionPhase, in this case they share only one function definition. If I rename it here they'd require having `nodes` and `executionNodes`. Or would you rename it on ExecutionPhase too?